boards/arm/efm32gg_slwstk6121a/doc/index.rst

```diff
@@ -0,0 +1,180 @@
+.. _efm32gg_slwstk6121a:
+
+WGM160P Starter Kit
+###################
+
+Overview
+********
+
+The WGM160P Starter Kit SLWSTK6121A comes with the BRD4321A radio board.
+This radio boards contains a WGM160P module, which combines the WF200 Wi-Fi
+transceiver with an EFM32GG11 microcontroller.
+
```

Comment:
Could you include a board image here (see other efm32 boards)? Maybe https://siliconlabs-h.assetsadobe.com/is/image//content/dam/siliconlabs/images/products/Bluetooth/wifi/wi-fi-modules/wgm160p-starter-kit.jpg? from https://www.silabs.com/development-tools/wireless/wi-fi/wgm160p-wifi-module-starter-kit

Reply:
Done.